indico/modules/events/papers/controllers/management.py

```diff
@@ -172,30 +172,47 @@ def _process(self):
 
 
 class RHManageReviewingSettings(RHManagePapersBase):
+    @property
+    def ratings(self):
+        return (PaperReviewRating.query
+                .join(PaperReviewRating.review)
+                .join(PaperReview.revision)
+                .join(PaperRevision._contribution)
+                .join(PaperReviewRating.question)
+                .filter(~Contribution.is_deleted,
+                        Contribution.event == self.event,
+                        PaperReviewQuestion.field_type == 'rating'))
+
+    def _scale_ratings(self, scale_min, scale_max):
+        prev_min = paper_reviewing_settings.get(self.event, 'scale_lower')
+        prev_max = paper_reviewing_settings.get(self.event, 'scale_upper')
+        if scale_min == prev_min and scale_max == prev_max:
+            return
+        assert scale_max > scale_min
+
+        paper_reviewing_settings.set(self.event, 'scale_lower', scale_min)
+        paper_reviewing_settings.set(self.event, 'scale_upper', scale_max)
```

Comment:
```suggestion
        paper_reviewing_settings.set_multi(self.event, {
            'scale_lower': scale_min,
            'scale_upper': scale_max,
        })
```

Reply:
Actually, I think this can completely go away since the `set_multi` right after the call to `self._scale_ratings()` should already save this...